client/src/ui/BitbakeCommands.ts

```diff
@@ -51,6 +51,7 @@ export function registerBitbakeCommands (context: vscode.ExtensionContext, bitba
     vscode.commands.registerCommand('bitbake.start-toaster-in-browser', async () => { await startToasterInBrowser(bitBakeProjectScanner.bitbakeDriver) }),
     vscode.commands.registerCommand('bitbake.stop-toaster', async () => { await stopToaster(bitBakeProjectScanner.bitbakeDriver) }),
     vscode.commands.registerCommand('bitbake.generate-dependency-graph', async (uri) => { await generateDependencyGraph(bitbakeWorkspace, bitBakeProjectScanner, uri) }),
+    vscode.commands.registerCommand('bitbake.examine-dependency-taskexp', async (uri) => { await examineDependenciesTaskexp(bitbakeWorkspace, bitBakeProjectScanner, uri) }),
```

Comment:
This command works fine! However I think most users will not be able to use taskexp to find what they want. The tool is very complex compared to oe-depends-dot. But it's rather an issue of the tool than your integration which works well.

Reply:
Well, I guess BitBake provides that tool because someone somewhere thinks it is useful. Lets not judge 🙃

I have in mind the command could be more interesting if we add an additional step where the user can select which ui they want to use. It would make it easier for the users to discover these tools and decide whether they find them useful or not.